Sources/SpeziViews/Views/MarkdownDocument/MarkdownDocumentView.swift

```diff
@@ -0,0 +1,84 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import MarkdownUI
+import SpeziFoundation
+import SwiftUI
+
+
+public struct MarkdownDocumentView<CustomElementView: View>: View {
+    public struct DividerRule {
+        @inlinable public static var never: Self {
+            .init { _, _ in false }
+        }
+        
+        @inlinable public static var always: Self {
+            .init { _, _ in true }
+        }
+        
+        @inlinable
+        public static func custom( // swiftlint:disable:this type_contents_order
+            _ shouldInsertDivider: @escaping @MainActor (_ fstBlockIdx: Int, _ sndBlockIdx: Int) -> Bool
```

Comment:
Maybe rather name these parameters as above and below? Applies to other parts of this file.

Reply:
Oh, sorry, meant that they should be named `blockAbove` and `blockBelow` instead if `fst` and `snd`, I would suggest to avoid abbreviations if we can and this makes more sense if we consider a divide which renders above and below an element.